scripts/end2end_examples.py

```diff
@@ -0,0 +1,128 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+To run this script you need to install papermill first:
+
+$ pip install papermill
+
+Then run the script from the argilla repo root:
+
+$ cd argilla-repo
+$ python scripts/end2end_examples.py --help
+"""
+
+import os
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import Dict, Optional
+
+import papermill
+import typer
+from argilla._constants import DEFAULT_API_KEY
+
+
+@dataclass
+class ExampleNotebook:
+    sort_index: int = field(init=False)
+    src_filename: Path
+    dst_filename: Path
+    parameters: Dict = field(default_factory=dict)
+
+    def __post_init__(self):
+        self.src_filename = Path(self.src_filename)
+        assert self.src_filename.exists(), f"File {self.src_filename} does not exist"
+        self.sort_index = int(self.src_filename.stem.split("-")[-1])
+        self.dst_filename = Path(self.dst_filename)
+        dst_folder = self.dst_filename.parent
+        if not dst_folder.exists():
+            dst_folder.mkdir(exist_ok=True)
+
+    def run(self):
+        try:
+            papermill.execute_notebook(
+                str(self.src_filename),
+                str(self.dst_filename),
+                parameters=self.parameters,
+                kernel_name="python3",
+            )
+            print(f"✅  {self.src_filename}")
```

Comment:
Does it make sense to align these prints with `rich`? I think we're already using it in other places. 

cc @davidberenstein1957

Reply:
I agree

Reply:
Do we have some place where we call the print from rich specifically? I've seen [this](https://github.com/argilla-io/argilla/blob/develop/src/argilla/cli/rich.py) for the CLI, but there's no specific print from `Console`. I just wanted some simple print to keep track of the process because it's for internal use, but let me know and I'll update it accordingly